doc/extdev/parserapi.rst

```diff
@@ -3,6 +3,35 @@
 Parser API
 ==========
 
+`The document of docutils describes`__ about the parsers like following:
+
+    The Parser analyzes the the input document and creates a node tree
```

Comment:
s/the the/the/

Reply:
It seems this is a typo of original text.
https://github.com/docutils-mirror/docutils/blob/master/docs/dev/hacking.txt#L69